File: java/sd/src/sd/Person.java
package sd;

//package สตั้ศý;

public class Person {
	private String name;
	private int age;
	private char sex;
	public Person(String name, int age, char sex) {
		super();
		this.name = name;
		this.age = age;
		this.sex = sex;
	}
	
	public Person() {
		super();
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}

	public char getSex() {
		return sex;
	}

	public void setSex(char sex) {
		this.sex = sex;
	}

	public String show(){
		return this.getName() + " " + this.getSex() + " " + this.getAge();
	}

}

File: c++/试验/实验八/person.cpp
#include"person.h"
#include<iostream>
using namespace std;

person::person(string I, string N,string S, int A)
{
	setId(I);
	setName(N);
	setSex(S);
	setAge(A);
}
void person::setId(string I)
{
	id = I;
}
void person::setName(string N)
{
	name = N;
}
void person::setSex(string S)
{
	sex = S;
}
void person::setAge(int A)
{
	age = A;
}
string person::getId() const
{
	return id;
}
string person::getName() const
{
	return name;
}
string person::getSex() const
{
	return sex;
}
int person::getAge() const
{
	return age;
}
void person::print()const
{
	cout << getId() << " "<< getName() << " " << getSex() << " " << getAge() << " years old";
}

person::~person()
{
}


